src/main/scala/sbtdatabricks/DatabricksPlugin.scala

```diff
@@ -183,8 +187,70 @@ object DatabricksPlugin extends AutoPlugin {
     }
   }
 
+  private lazy val executeCommandImpl: Def.Initialize[Task[Seq[CommandStatus]]] = Def.task {
+    val client = dbcApiClient.value
+    val language = dbcExecutionLanguage.value
+    val onClusters = dbcClusters.value
+    val (allClusters, _) = dbcFetchClusters.value
+    val commandFile = dbcCommandFile.value
+    val commandStatuses = Seq.empty[CommandStatus]
+
+    @annotation.tailrec
+    def onContextCompletion(contextId: ContextId, cluster: Cluster) : Option[ContextId] = {
+      val contextStatus = client.checkContext(contextId, cluster)
+      contextStatus.status match {
```

Comment:
You might not need the `.status`. Could you please check? If we don't need them, you may remove them from `DBCContext` and `DBCLanguage`